### PR TITLE
Feat: Simplify action now that we have a Github App

### PR DESCRIPTION
**Pull request by slaterb1**
# Overview
Main flow of events is handled via The Ops Platform Workflow Agent Github App. This update extends the insights returned by the Github App by allowing users to specify additional events they care about and deployments.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,67 +1,63 @@
 # CTO.ai GitHub Action
 
-The CTO.ai GitHub Action makes it easy to integrate GitHub into the CTO.ai
-Delivery Metrics platform.
+This Github Action allows you to extend the insights data received by the Ops Platform Github App. You can add data to you workflows for deployment tools that are not configured with Github's deployment API.
```
Comment (kc-dot-io) [inline README.md]: Let's remove "the Ops Platform" and just say "CTO.ai".

File: README.md [hunk 2]
```diff
@@ -1,67 +1,63 @@
...
 
-## CTO.ai Drop-In GitHub Action Agent
+## Inputs
 
-The CTO.ai Drop-In GitHub Agent will automatically record some CTO.ai Delivery
-& LifeCycle Metrics that can then be viewed on the CTO.ai Delivery Metrics
-dashboard.
+### `token`
+**Required** Ops Platform api token, follow [these docs to generate](https://cto.ai/docs/integrate-any-tool)
```
Comment (kc-dot-io) [inline README.md]: Let's remove "the Ops Platform" and just say "CTO.ai".

File: README.md [hunk 3]
```diff
@@ -1,67 +1,63 @@
...
 
-### Install Agent
+### `team_id`
+**Required** Ops Platform team ID sending the data, follow [these docs to access](https://cto.ai/docs/integrate-any-tool)
```
Comment (kc-dot-io) [inline README.md]: Let's remove "the Ops Platform" and just say "CTO.ai".

**Review by lucamaraschi — APPROVED** [review]
Code looks really good

**Review by kc-dot-io — CHANGES_REQUESTED** [review]

**Comment by kc-dot-io** [comment]
All the functionality LGTM (big improvement here), couple cosmetic comments about branding.

**Review by kc-dot-io — APPROVED** [review]

**Pull request merged**